util/new_keyboard.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+import argparse
+import fileinput
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+from distutils.dir_util import copy_tree
+
+def print_error(message):
+    print("[\033[0;91mERROR\033[m] " + message, file=sys.stderr)
+
+def prompt(message, default=""):
+    """Ask the user for input, showing the default value in brackets."""
+    if default:
+        message += f" [{default}]"
+    value = input(message + ": ")
+    return value or default
+
+def get_git_username():
+    """Return the username configured in git config, or None."""
+    try:
+        return subprocess.check_output(['git', 'config', '--get', 'user.name'], encoding='utf-8').rstrip('\r\n')
```

Comment:
I don't recommend going past 80-[ish](https://www.youtube.com/watch?v=wf-BqAjZb8M&t=260) columns, it's not readable. Use your preferred way of breaking up expressions into multiple lines, but I suggest this:
```suggestion
        return subprocess.check_output(
            ['git', 'config', '--get', 'user.name'],
            encoding='utf-8',
        ).rstrip('\n')
```
Or you can do it the way [black](https://github.com/ambv/black#call-chains) does it.

Also, `subprocess.run()`, `check_output()` etc. [normalize](https://docs.python.org/3/library/subprocess.html#frequently-used-arguments) newlines to `'\n'`, so `'\r'` isn't necessary.

Reply:
I'm not a fan of line wrapping, since I have a perfectly good scroll wheel, a decently sized monitor (not 4K) and I don't code in a terminal, but if it's part of PEP8...